third_party/txt/src/txt/paragraph.h

```diff
@@ -193,6 +193,7 @@ class Paragraph {
   std::shared_ptr<FontCollection> font_collection_;
 
   minikin::LineBreaker breaker_;
+  std::unique_ptr<icu::BreakIterator> word_breaker_;
```

Comment:
Do you think it'd make sense to declare this `mutable` and leave `GetWordBoundary` const?

Reply:
done